src/libsync/iconjob.cpp

```diff
@@ -31,11 +31,13 @@ IconJob::IconJob(const QUrl &url, QObject *parent) :
 
 void IconJob::finished(QNetworkReply *reply)
 {
-    if (reply->error() != QNetworkReply::NoError)
-        return;
-
     reply->deleteLater();
     deleteLater();
+
+    if (reply->error() != QNetworkReply::NoError) {
```

Comment:
I do not really understand this change
how an user of `IconJob` knows that there has been an error ?
that would be needed to have a placeholder image in place of the real one in case of an error

Reply:
This change was just to make sure the job gets deleted if there is an error. Because from my understanding, at the moment, the job gets only deleted if the job succeeds.